controllers/topics.controller.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res) => {
+  fetchTopics().then((body) => {
+    res.status(200).send(body);
```

Comment:
We should always send back a response object with a specific key, in this case it should be topics

Reply:
Thank you! Have now actioned this :)